src/resources/material.js

```diff
@@ -140,14 +140,14 @@ class MaterialHandler {
     }
 
     _assignTexture(parameterName, materialAsset, texture) {
-        materialAsset.data[parameterName] = texture;
+        // NB removed swapping out asset id for resource here
         materialAsset.resource[parameterName] = texture;
     }
 
     // assign a placeholder texture while waiting for one to load
     // placeholder textures do not replace the data[parameterName] value
     // in the asset.data thus preserving the final asset id until it is loaded
-    _assignPlaceholderTexture(parameterName, materialAsset) {
+    _getPlaceholderTexture(parameterName, materialAsset) {
```

Comment:
seems like this comment is no longer valid? please update it with latest usage

Reply:
I'll refactor the comment

Reply:
done